sdk/core/azure-core/tests/testserver_tests/coretestserver/coretestserver/files/test_with_header.tar.gz

```diff
@@ -0,0 +1 @@
+test
```

Comment:
We can use the same file that's added in this PR: https://github.com/Azure/azure-sdk-for-python/pull/22030/files